docker/adminrouter-upstreams.conf

```diff
@@ -7,6 +7,11 @@ proxy_ssl_verify_depth 5;
 # For the sake of tests simplicity, we will not be using/testing leader.mesos
 # alias here. This should be done by integration tests.
 set $upstream_mesos http://127.0.0.2:5050;
-set $upstream_marathon http://127.0.0.1:8443;
+set $upstream_marathon http://127.0.0.1:8080;
```

Comment:
So, `http://127.0.0.1:8443` was actually wrong as of mismatch between scheme and port, yes?